real_time/surface_src/java_src/Alice/build.gradle

```diff
@@ -50,7 +50,8 @@ dependencies {
     compile files('library/orsoncharts-1.4-eval-nofx.jar')
 
 
-    compile files('library/RXTXcomm.jar')
+    compile files('library/purejavacomm.jar')
```

Comment:
I think it would be best if you could find these in a maven repo somewhere, so that we don't need to keep the `.jar` files on hand --> see [this tutorial][1] for a little more info on maven repos, and [the developer's maven integration][2] for the dependency

[1]:https://docs.gradle.org/current/userguide/artifact_dependencies_tutorial.html#sec:declaring_your_dependencies
[2]:https://github.com/nyholku/purejavacomm/wiki/Installing-PureJavaComm#using-pjc-with-maven